packages/FormElement/stories/examples/AlternateLayouts.js

```diff
@@ -331,20 +332,57 @@ const ExampleStory = () => {
           </span>
         </FormElement.Instructions>
         <FormElement.Content>
-          {({ idForLabel, ariaDescribedBy }) => (
-            <Input
-              id={idForLabel}
-              onChange={handleChange}
-              value={value}
-              placeholder="Form placeholder"
-              aria-describedby={ariaDescribedBy}
-              aria-required={hasRequiredLabel}
-              hasError={Boolean(errorText.length)}
-              isDisabled={isDisabled}
-              isReadOnly={isReadOnly}
-              size={size}
-            />
-          )}
+          <Input
+            onChange={handleChange}
+            value={value}
+            placeholder="Form placeholder"
+            aria-required={hasRequiredLabel}
+            hasError={Boolean(errorText.length)}
+            isDisabled={isDisabled}
+            isReadOnly={isReadOnly}
+            size={size}
+          />
+        </FormElement.Content>
+        <FormElement.Help>
+          Give me some help. <a href="wegalvanize.com">Learn more</a>.
+        </FormElement.Help>
+      </FormElement>
+      <Rule />
+      <Tagline>Form Element overriding label association with child</Tagline>
+      <br />
+      <FormElement
+        hasRequiredLabel={hasRequiredLabel}
+        label={
+          <Heading level={5}>
+            <strong>
+              <i>Form Label</i>
+            </strong>
+          </Heading>
+        }
+      >
+        <FormElement.Instructions>Some instructions</FormElement.Instructions>
+        <FormElement.Content>
+          <Input
+            onChange={handleChange}
+            value={value}
+            placeholder="Form placeholder"
+            aria-required={hasRequiredLabel}
+            hasError={Boolean(errorText.length)}
+            isDisabled={isDisabled}
+            isReadOnly={isReadOnly}
+            size={size}
+          />
+          <Input
+            onChange={handleChange}
+            value={value}
+            placeholder="Form placeholder"
+            aria-required={hasRequiredLabel}
+            hasError={Boolean(errorText.length)}
+            isDisabled={isDisabled}
+            isReadOnly={isReadOnly}
+            size={size}
+            overridelabelassociation="true" // Add to override default label association
```

Comment:
this is a proposal for how we can override which element is associated with the label, perhaps wrapping with a `FormElement.AssociateLabel` component could be another approach, or any better proposals?